.github/.wordlist.txt

```diff
@@ -724,6 +725,7 @@ JLink
 JLinkExe
 JLinkRTTClient
 JN
+joaompinto
```

Comment:
could you set the link as a wiki URL and then it may be ignored by our spellchecker?

This is for sure not a `word` and I don't thik we should spellcheck URLs.

Reply:
Sure, no problem. I will get this adjusted 👍🏼

Reply:
Fixed in latest commit